spring-cloud-netflix-core/src/test/java/org/springframework/cloud/netflix/zuul/FormZuulServletProxyApplicationTests.java

```diff
@@ -65,7 +65,7 @@
 @RunWith(SpringJUnit4ClassRunner.class)
 @SpringApplicationConfiguration(classes = FormZuulServletProxyApplication.class)
 @WebAppConfiguration
-@IntegrationTest({ "server.port:0", "zuul.routes.simple:/zuul/simple/**" })
+@IntegrationTest({ "server.port:0", "zuul.routes.simple:/simple/**" })
```

Comment:
The context is the big files transfer through Zuul.
See https://github.com/spring-cloud/spring-cloud-netflix/issues/254#issuecomment-83958338

My problem is that the ```/zuul``` prefix is not set in the ProxyRouteLocator, meaning it will not be stripped from the path, so no path will be found.

The changed was written gfor this scenarion, but in my opinion the tested route should be declared without the ```/zuul``` prefix.